HeroLib/Events/Main.lua

```diff
@@ -373,15 +373,16 @@ HL:RegisterForEvent(function(Event, Arg1)
   end
 
   -- Refresh Artifact
-  if Event == "PLAYER_LOGIN"
-    or (Event == "PLAYER_EQUIPMENT_CHANGED" and Arg1 == 16)
-    or PrevSpec ~= Cache.Persistent.Player.Spec[1] then
-    Spell:ArtifactScan()
-  end
+  --if Event == "PLAYER_LOGIN"
+  --  or (Event == "PLAYER_EQUIPMENT_CHANGED" and Arg1 == 16)
+  --  or PrevSpec ~= Cache.Persistent.Player.Spec[1] then
+  --  Spell:ArtifactScan()
+  --end
 
   -- Load / Refresh Core Overrides
   if Event == "PLAYER_LOGIN" then
     C_Timer.After(3, function() HL.LoadOverrides(Cache.Persistent.Player.Spec[1]) end) -- TODO: fix timing issue via event?
+    Unit.Player:Cache()
```

Comment:
I'm not sure we have all infos at this moment (especially on some slow PC), maybe it might be better to delay it like the line above.
There is an event that guarantee that though, just need to find it.